README.rst

```diff
@@ -332,3 +332,11 @@ Root Cause: SignalFx's API endpoints (``api.signalfx.com``,
 the ``urllib3`` module in Python versions prior to 2.7.8 had a bug that causes
 the above issue. This was fixed in later versions of Python; we recommend using
 Python 2.7.9 or newer when using this library.
+
+Exceeding int64 integer sizes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The protocol buffer used to transmit data through the ingest API restricts 
+integers and floats to (-9223372036854775808 - 9223372036854775807).  long 
```

Comment:
@mdubbyap I actually meant to say "integers and longs" it was a typo on my part.  Should I try to enforce a check for floats given the imprecision?